src/domain/events/eventActionsDropdown/EventActionsDropdown.tsx

```diff
@@ -64,6 +65,18 @@ const EventActionsDropdown: React.FC<EventActionsDropdownProps> = ({
     navigate(`/${locale}${ROUTES.CREATE_EVENT}`);
   };
 
+  const sendEmail = () => {
+    let targetEmail = '';
+    if (event.createdBy != null && event.createdBy != undefined) {
+      targetEmail = event.createdBy.split('-')[1].trim();
+    }
+    window.location.href =
+      'mailto:' +
+      targetEmail +
+      '?subject=' +
+      getLocalisedString(event.name, locale);
+  };
```

Comment:
Could add a unit test to test that sendEmail + window.location.href functions are called with correct values.